omnibus.gemspec

```diff
@@ -23,9 +23,9 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency 'chef-sugar',       '~> 3.0'
   gem.add_dependency 'cleanroom',        '~> 1.0'
-  gem.add_dependency 'mixlib-shellout',  '~> 1.4'
+  gem.add_dependency 'mixlib-shellout',  '~> 2.0'
   gem.add_dependency 'mixlib-versioning'
-  gem.add_dependency 'ohai',             '~> 7.2'
+  gem.add_dependency 'ohai',             '~> 8.2'
   gem.add_dependency 'ruby-progressbar', '~> 1.7'
```

Comment:
I'm a little worried about bump the major versions of both `mixlib-shellout` AND `ohai`. We'll need to do some pretty heavy testing that this doesn't break things on all the other platforms we build for.